src/shared/components/setup-locker/SetupLocker.js

```diff
@@ -1,52 +1,63 @@
 import React, { Component } from 'react';
+import { FlowModal, FlowModalStep, setAppElement } from '@nomios/web-uikit';
 import PropTypes from 'prop-types';
 
-import SetupMasterLock from './steps/SetupMasterLock';
-import SetupIdleTimer from './steps/SetupIdleTimer';
+import SetPassphrase from './setup-locker-steps/set-passphrase/SetPassphrase';
+import SetTimeout from './setup-locker-steps/set-timeout/SetTimeout';
+
+setAppElement('#root');
```

Comment:
Move this to `src/index.js` right above `ReactDOM.render(<App />, document.getElementById('root'));`